tools/mo/openvino/tools/mo/middle/permute_tensor_iterator.py

```diff
@@ -197,10 +198,16 @@ def find_ports(port_map: list, attrs: dict):
 
         isomorphism['input_unsqueezed_i'].shape = isomorphism['input_unsqueezed_i'].shape[[1, 0, 2]]
         isomorphism['input_unsqueezed_i'].infer(isomorphism['input_unsqueezed_i'])
-        isomorphism['squeeze_dim'].value = ti.input_port_map[data_input_port]['axis']
-        isomorphism['squeeze_dim'].infer(isomorphism['squeeze_dim'])
-        isomorphism['squeeze']['need_shape_inference'] = True
+        squeeze = isomorphism['squeeze']
+        squeeze_dim = Const(squeeze.graph, {'value': ti.input_port_map[data_input_port]['axis'],
+                                            'need_shape_inference': True,
+                                            'override_output_shape': True}).create_node()
+        squeeze.in_port(1).get_connection().set_source(squeeze_dim.out_port(0))
+        squeeze['need_shape_inference'] = True
 
-        isomorphism['unsqueeze_dim'].value = ti.output_port_map[data_output_port]['axis']
-        isomorphism['unsqueeze_dim'].infer(isomorphism['unsqueeze_dim'])
-        isomorphism['unsqueeze'].infer(isomorphism['unsqueeze'])
+        unsqueeze = isomorphism['unsqueeze']
+        unsqueeze_dim = Const(unsqueeze.graph, {'value': ti.output_port_map[data_output_port]['axis'],
+                                                'need_shape_inference': True,
+                                                'override_output_shape': True}).create_node()
+        unsqueeze.in_port(1).get_connection().set_source(unsqueeze_dim.out_port(0))
+        unsqueeze['need_shape_inference'] = True
```

Comment:
Why do you change `Squeeze` and `Unsqueeze` nodes here?

Reply:
I checked, didn't see any problems there.